ironic/drivers/modules/drac/raid.py

```diff
@@ -635,7 +715,7 @@ def _filter_logical_disks(logical_disks, include_root_volume,
     return filtered_disks
 
 
-def _commit_to_controllers(node, controllers):
+def _commit_to_controllers(node, controllers, substep=None):
```

Comment:
Instead of defaulting substep to None, let's default it to "complete"

Reply:
Done